insights/config/specs.py

```diff
@@ -388,9 +388,11 @@
     "systemid"                  : First([SimpleFileSpec("etc/sysconfig/rhn/systemid"),
                                     SimpleFileSpec("conf/rhn/sysconfig/rhn/systemid")]),
     "teamdctl_state_dump"       : CommandSpec("/usr/bin/teamdctl {iface} state dump", iface=r"team\S+"),
-    "tmpfilesd"               : All([PatternSpec("etc/tmpfiles.d/.*\.conf"),
+    "tmpfilesd"                 : All([PatternSpec("etc/tmpfiles.d/.*\.conf"),
                                     PatternSpec(r"usr/lib/tmpfiles.d/.*\.conf"),
                                     PatternSpec(r"run/tmpfiles.d/.*\.conf")]),
+    "tomcat_server_log"         : All([PatternSpec("usr/share/tomcat/logs/catalina.*\.log"),
```

Comment:
Since the new `tomcat_server_log` will cover the old `catalina.out` (correct me if it's wrong plz), I suggest removing the old one in the master branch and mark it as deprecated (see #733) in the 1.x branch.